bundles/org.openhab.binding.monopriceaudio/src/main/java/org/openhab/binding/monopriceaudio/internal/handler/MonopriceAudioHandler.java

```diff
@@ -0,0 +1,706 @@
+                                            connector.sendCommand(MonopriceAudioZone.valueOf(zoneName),
+                                                    MonopriceAudioCommand.SOURCE, value);
+                                        } catch (MonopriceAudioException e) {
+                                            logger.warn("Error Setting Source for  All Zones: {}", e.getMessage());
+                                        }
+                                    }
+                                });
+                            }
+                        }
+                        break;
+                    case CHANNEL_TYPE_ALLVOLUME:
+                        if (command instanceof PercentType) {
+                            int value = (int) Math
+                                    .round(((PercentType) command).doubleValue() / 100.0 * (MAX_VOLUME - MIN_VOLUME))
+                                    + MIN_VOLUME;
+                            allVolume = value;
+                            zoneStream.forEach((zoneName) -> {
+                                if (!ignoreZones.contains(zoneName)) {
+                                    try {
+                                        connector.sendCommand(MonopriceAudioZone.valueOf(zoneName),
+                                                MonopriceAudioCommand.VOLUME, value);
+                                    } catch (MonopriceAudioException e) {
+                                        logger.warn("Error Setting Volume for All Zones: {}", e.getMessage());
+                                    }
+                                }
+                            });
+                        }
+                        break;
+                    case CHANNEL_TYPE_ALLMUTE:
+                        if (command instanceof OnOffType) {
+                            int cmd = command == OnOffType.ON ? 1 : 0;
+                            zoneStream.forEach((zoneName) -> {
+                                if (!ignoreZones.contains(zoneName)) {
+                                    try {
+                                        connector.sendCommand(MonopriceAudioZone.valueOf(zoneName),
+                                                MonopriceAudioCommand.MUTE, cmd);
+                                    } catch (MonopriceAudioException e) {
+                                        logger.warn("Error Setting Mute for All Zones: {}", e.getMessage());
+                                    }
+                                }
+                            });
+                        }
+                        break;
+                    default:
+                        success = false;
+                        logger.debug("Command {} from channel {} failed: unexpected command", command, channel);
+                        break;
+                }
+
+                if (success) {
+                    logger.debug("Command {} from channel {} succeeded", command, channel);
```

Comment:
Success is expected, so I think you can change the logging level to trace.
```suggestion
                    logger.trace("Command {} from channel {} succeeded", command, channel);
```